packages/code-studio/src/main/AppMainContainer.tsx

```diff
@@ -756,23 +756,22 @@ export class AppMainContainer extends Component<
               className="ml-1"
             />
             <div>
-              <button type="button" className="btn btn-link btn-panels-menu">
-                <FontAwesomeIcon icon={dhShapes} />
+              <Button kind="ghost" className="btn-panels-menu" icon={dhShapes}>
```

Comment:
As mentioned in your comment, this is complaining because no `onClick` is provided.
The reason no `onClick` is provided is because `AppControlsMenu`/`DropdownMenu` adds an onclick listener to it's parent: https://github.com/deephaven/web-client-ui/blob/75204eda3d72f91ad80215af6177927ae1bea1ac/packages/components/src/menu-actions/DropdownMenu.tsx#L92
This paradigm that `DropdownMenu` uses is not very React-y at all - I thought it was clever at the time, but it's stupid.
Changing all instances of `DropdownMenu` does not make sense right now, so I'd say just leave that and then just add an empty `onClick` handle to Button here so it works:
 ```suggestion
              <Button
                kind="ghost"
                className="btn-panels-menu"
                icon={dhShapes}
                onClick={() => {
                  // no-op: click is handled in `AppControlsMenu` (which uses a `DropdownMenu`)
                }}
              >
```